semeio/workflows/misfit_preprocessor/misfit_preprocessor.py

```diff
@@ -76,7 +74,5 @@ def _get_observations(facade):
 @hook_implementation
 def legacy_ertscript_workflow(config):
     workflow = config.add_workflow(MisfitPreprocessorJob, "MISFIT_PREPROCESSOR")
-    _schema = misfit_preprocessor.config._SCHEMA
-    rst_doc = configsuite.docs.generate(_schema)
-    workflow.description = rst_doc
+    workflow.description = "No description as of now"
```

Comment:
Maybe use pydantic schema: https://pydantic-docs.helpmanual.io/usage/schema/

Reply:
Yes, though I would like to avoid creating my own `rst` from the schema. Have you found a good way of doing that?

Reply:
Maybe something like https://pypi.org/project/sphinx-jsonschema/ ?

Reply:
The problem with most of these packages is that they are sphinx plugins, while what we are doing in this case is passing an `rst` string to `ert` for it to create the actual documentation.

Reply:
Could the consumer of the documentation be required to understand the sphinx extension?

Reply:
There is also https://github.com/inspirehep/jsonschema2rst which is not well maintained but does the job.

Reply:
It looks like it could do the job, though the non-cli entry point is not documented.

Reply:
Then, I guess your solution is the best we can do for now.